docs/gettingstarted.md

```diff
@@ -69,6 +69,8 @@ The roles controls access to the service API.
 | `url` | The service key URL pointing to the service instance | `https://service.key.url.com/2787d5ea-17cb-4c71-85ca-a9600d4b97c9`|
 | `apiKey`| The service key API key. This key is generated by Mobile Services | `apiKey`|
 
+Use Cloud Foundry Bindings for X509 service keys.
```

Comment:
It would not really be what we call services keys. It's a binding.

Reply:
Right, but https://help.sap.com/doc/f53c64b93e5140918d676b927a3cd65b/Cloud/en-US/docs-en/guides/features/security/admin/service-keys.html documents them as X509 service keys.

Reply:
I don't understand what this sentence is supposed to mean. Should it encourage user to use `X.509` when creating service-key instead of the default `API Key` option?